core/src/main/scala/kafka/server/KafkaApis.scala

```diff
@@ -2529,8 +2529,8 @@ class KafkaApis(val requestChannel: RequestChannel,
   }
 
   def ensureInterBrokerVersion(version: MetadataVersion): Unit = {
-    if (config.interBrokerProtocolVersion.isLessThan(version))
-      throw new UnsupportedVersionException(s"inter.broker.protocol.version: ${config.interBrokerProtocolVersion} is less than the required version: ${version}")
+    if (metadataCache.metadataVersion().isLessThan(version))
+      throw new UnsupportedVersionException(s"inter.broker.protocol.version: ${metadataCache.metadataVersion()} is less than the required version: ${version}")
```

Comment:
Should we change `inter.broker.protocol.version` accordingly?

Reply:
Thanks for your comment, I will modify this.Should I use the `metadata.version` is clear enough?